eZ/Bundle/EzPublishCoreBundle/Imagine/BinaryLoader.php

```diff
@@ -56,6 +57,17 @@ public function find($path)
             );
         } catch (NotFoundException $e) {
             throw new NotLoadableException("Source image not found in $path", 0, $e);
+        } catch (InvalidBinaryFileIdException $e) {
+            $message =
+                "Source image not found in $path. Repository images path are expected to be " .
+                "relative to the var/<site>/storage/images directory";
+
+            $suggestedPath = preg_replace('#var/[^/]+/storage/images/#', '', $path);
+            if ($suggestedPath) {
```

Comment:
http://php.net/manual/en/function.preg-replace.php says: 

> If matches are found, the new subject will be returned, otherwise subject will be returned unchanged or NULL if an error occurred.

So is "subject not found" an error? Seems there are cases where the subject will be returned unchanged, which would result in a suggestion to repeat the same command...?